experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -342,13 +342,22 @@ export const getRequestInfo = (
  * Makes sure options is of type string or object
```

Comment:
You may want to update this function description to better represent the new logic added in this PR.

Reply:
Okay, thanks

I will do that